cmd/cosign/cli/sign/privacy/privacy.go

```diff
@@ -19,12 +19,12 @@ import "sync"
 const (
 	// spacing is intentional to have this indented
 	Statement = `
-	Note that there may be personally identifiable information associated with this signed artifact.
-	This may include the email address associated with the account with which you authenticate.
-	This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.
-`
-	StatementConfirmation = "By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs."
-)
+	The Cosign service, hosted by sigstore a Series of LF Projects, LLC, is provided pursuant to the Hosted Project Tools Terms of Use, available at https://lfprojects.org/policies/hosted-project-tools-terms-of-use/.
+	Note that if your submission includes personal data associated with this signed artifact, it will be part of an immutable record.
+	This may include the email address associated with the account with which you authenticate your attestation and contractual Agreement.
```

Comment:
Would recommend deleting "your attestation and contractual agreement" - Not all entries are attestations, and I'm not sure contractual agreement makes sense in this context.